Python/Product/PythonTools/PythonTools/Project/PythonNotSupportedInfoBar.cs

```diff
@@ -0,0 +1,95 @@
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.PythonTools.Project {
+    internal class PythonNotSupportedInfoBar : PythonInfoBar {
+        private Func<IPythonInterpreterFactory> _getActiveInterpreterFunc;
+        private IPythonInterpreterFactory _activeInterpreter { get { return _getActiveInterpreterFunc(); } }
+
+        private GeneralOptions _options;
+        private String _context;
+
+        private string _moreInformationLink = @"https://github.com/microsoft/PTVS/issues/1"; //TODO Raymon  Change URL
+        private readonly Version _pythonVersionNotSupported = new Version("3.8");
+        private bool _infoBarTriggered = false;
+
+        public PythonNotSupportedInfoBar(IServiceProvider site, string context, Func<IPythonInterpreterFactory> activeInterpreter) : base(site) {
+            _options = Site.GetPythonToolsService().GeneralOptions;
+            _getActiveInterpreterFunc = activeInterpreter;
+            _context = context;
+        }
+
+        public override Task CheckAsync() {
+            if (IsCreated ||
+                !_options.PromptForPythonVersionNotSupported ||
+                _infoBarTriggered ||
+                _activeInterpreter.Configuration.Version < _pythonVersionNotSupported
+            ) {
+                return Task.CompletedTask;
+            }
+
+            var infoBarTextSpanMessage = new InfoBarTextSpan(Strings.PythonVersionNotSupportInfoBarText.FormatUI(_activeInterpreter.Configuration.Version));
+            var infoBarMessage = new List<IVsInfoBarTextSpan>() { infoBarTextSpanMessage };
+            var actionItems = new List<InfoBarActionItem>() {
+                new InfoBarHyperlink(Strings.PythonVersionNotSupportMoreInfo, (Action)MoreInformationAction),
+                new InfoBarHyperlink(Strings.PythonVersionNotSupportedDontShowMessageAgain, (Action)DoNotShowAgainAction)
+            };
+
+            LogEvent(PythonVersionNotSupportedInfoBarAction.Prompt);
+            Create(new InfoBarModel(infoBarMessage, actionItems, KnownMonikers.StatusInformation));
+            _infoBarTriggered = true;
+
+            return Task.CompletedTask;
+        }
+
+        private void MoreInformationAction() {
+            LogEvent(PythonVersionNotSupportedInfoBarAction.MoreInfo);
+            Close();
+
+            VsShellUtilities.OpenBrowser(_moreInformationLink);
+        }
+
+        private void DoNotShowAgainAction() {
+            LogEvent(PythonVersionNotSupportedInfoBarAction.Ignore);
+            Close();
+
+            _options.PromptForPythonVersionNotSupported = false;
+            _options.Save();
+        }
+
+        private void LogEvent(string action) {
+            Logger?.LogEvent(
+                PythonLogEvent.PythonNotSupportedInfoBar,
+                new PythonVersionNotSupportedInfoBarInfo() {
+                    Action = action,
+                    Context = _context,
+                    PythonVersion = _activeInterpreter.Configuration.Version
```

Comment:
this re-evaluates active interpreter, which could now be different than the one that triggered the info bar (user switch to python 3.8, info bar appears, switch back to 3.7, then clicks on more info)

I guess this could be avoided if you were to cache the active interpreter when `_infoBarTriggered` and use that instead.